docs/en/latest/terminology/global-rule.md

```diff
@@ -32,6 +32,12 @@ A [Plugin](./plugin.md) configuration can be bound directly to a [Route](./route
 
 Compared with the plugin configuration in Route, Service, Plugin Config, and Consumer, the plugin in the Global Rules is always executed first.
 
+::: note
+
+For stream routes, only plugins that run on `before_proxy` [phase](../terminology/plugin.md#plugins-execution-order) get executed as Global Rules.
```

Comment:
@shreemaan-abhishek please revert this note. We can still merge the other changes.